app/models/concerns/alegre_webhooks.rb

```diff
@@ -0,0 +1,23 @@
+class AlegreCallbackError < StandardError
+end
+module AlegreWebhooks
+  extend ActiveSupport::Concern
+  module ClassMethods
+    def webhook(request)
+      #at this point, we know that the data is stored and ready for querying, so we can safely query based on package of:
+      # return requests.post(f"{check_api_host}/api/webhooks/alegre", json={
+      #     "action": action,
+      #     "model_type": model_type,
+      #     "data": data,
+      # })
+      begin
+        doc_id = request.params["data"]["requested"]["body"]["id"]
+        redis = Redis.new(REDIS_CONFIG)
+        redis.lpush(doc_id, request.params.to_json)
```

Comment:
Let's add a prefix to the Redis keys, just for better scoping/debugging if needed, like a namespace... so, instead of `doc_id`, something like `"alegre:webhook:#{doc_id}"`, for example.

Reply:
works for me!